manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -31,6 +31,11 @@ type ReadModuleArgs struct {
 	// +required
 	Source DataStore `json:"source"`
 
+	// AssetName represents the asset name to be used for accessing the data when it is ready
+	// It is copied from the M4DApplication resource
+	// +required
+	AssetName string `json:"assetName"`
```

Comment:
Already approved, but we might want to use `AssetID` to be consistent?

Reply:
Changed the name.